tests/test_conf.py

```diff
@@ -158,6 +179,32 @@ def test_load_empty_pyproject_toml_and_cz_toml_with_config(_, tmpdir):
             cfg = config.read_cfg()
             assert cfg.settings == _settings
 
+    def test_load_pyproject_toml_not_in_root_folder(_, tmpdir):
```

Comment:
```suggestion
    def test_load_pyproject_toml_from_config_argument(_, tmpdir):
```

this name can be applied to other test cases